apple/RNCWebView.mm

```diff
@@ -74,10 +76,10 @@ - (instancetype)initWithFrame:(CGRect)frame
                     .lockIdentifier = [[dictionary valueForKey:@"lockIdentifier"] doubleValue],
                     .title = std::string([[dictionary valueForKey:@"title"] UTF8String]),
                     .navigationType = stringToOnShouldStartLoadWithRequestNavigationTypeEnum(std::string([[dictionary valueForKey:@"navigationType"] UTF8String])),
-                    .canGoBack = [[dictionary valueForKey:@"canGoBack"] boolValue],
-                    .canGoForward = [[dictionary valueForKey:@"canGoBack"] boolValue],
-                    .isTopFrame = [[dictionary valueForKey:@"isTopFrame"] boolValue],
-                    .loading = [[dictionary valueForKey:@"loading"] boolValue],
+                    .canGoBack = static_cast<bool>([[dictionary valueForKey:@"canGoBack"] boolValue]),
+                    .canGoForward = static_cast<bool>([[dictionary valueForKey:@"canGoBack"] boolValue]),
+                    .isTopFrame = static_cast<bool>([[dictionary valueForKey:@"isTopFrame"] boolValue]),
+                    .loading = static_cast<bool>([[dictionary valueForKey:@"loading"] boolValue]),
```

Comment:
Fixes `Non-constant-expression cannot be narrowed from type 'BOOL' (aka 'signed char') to 'bool' in initializer list` when building for Release